src/components/LocationPage/ChartsHolder.tsx

```diff
@@ -60,9 +60,14 @@ const ChartsHolder = (props: {
   const theme = useTheme();
   const isMobile = useMediaQuery(theme.breakpoints.down('xs'));
 
-  const { pathname } = useLocation();
+  const { pathname, hash } = useLocation();
   const isRecommendationsShareUrl = pathname.includes('recommendations');
 
+  const defaultMetric =
+    hash === '#explore-chart'
+      ? ExploreMetric.HOSPITALIZATIONS
+      : ExploreMetric.CASES;
```

Comment:
FWIW- We actually have existing navigation / scrolling set up for URLs like https://covidactnow.org/us/wa/explore/hospitalizations (this still works from the original version of explore chart sharing), so if we thought it was useful we could leverage that.  But since you have this working, I assume it makes sense to just ship it!